src/components/PlayerList.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './playerlist.css'
+import Player from './Player';
+
+const PlayerList = ( props ) => (
+    <React.Fragment>
+            <table>
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Position</th>
+                    <th>Jersey Number</th>
+                    <th>Nacionality</th>
+                </tr>
+            </thead>
+            </table>
+                        {
+                        props.players.map(player=>(
+                            <Player
+                                key={player.id}
+                                name={player.name}
+                                position={player.position}
+                                jerseyNumber={player.jerseyNumber}
+                                nationality={player.nationality}
+                            />
+                        ))
+                
+                        }
+    </React.Fragment>

```

Comment:
Minor, but this could be: 
```suggestion
       <table>
            <thead>
                <tr>
                    <th>Name</th>
                    <th>Position</th>
                    <th>Jersey Number</th>
                    <th>Nacionality</th>
                </tr>
            </thead>
            <tbody>
                   {
                        props.players.map(player=>(
                            <Player
                                key={player.id}
                                name={player.name}
                                position={player.position}
                                jerseyNumber={player.jerseyNumber}
                                nationality={player.nationality}
                            />
                        ))
                    }
            </tbody>
       </table>
```

Reply:
done

Reply:
you are returning a single table now :) you don't need the fragment